### ISSUE TITLE
Restore 2.47 - ERROR: [029]: raised from local-13 protocol: zst error: [-10] Unknown frame descriptor

### ISSUE THREAD

**Issue by fmbiete** [pgbackrest/pgbackrest#2190]
Please provide the following information when submitting an issue (feature requests or general comments can skip this):

1. pgBackRest version: 2.46 and 2.47

3. PostgreSQL version: 13.11

4. Operating system/version - if you have more than one server (for example, a database server, a repository host server, one or more standbys), please specify each:
RHEL 9.2 x86_64
Replication cluster of 3 servers - Primary + 2 physical replication standby
pgbackrest - standby backup enabled, compression zstd for backupset, incremental block backup
Backups were created with 2.46, I have tried to restore with 2.46 and 2.47 with the same error

6. Did you install pgBackRest from source or from a package?
Built from source

7. Please attach the following as applicable:
    - `pgbackrest.conf` file(s)
    - `postgresql.conf` settings applicable to pgBackRest (`archive_command`, `archive_mode`, `listen_addresses`, `max_wal_senders`, `wal_level`, `port`)
    - errors in the postgresql log file before or during the time you experienced the issue
    - log file in `/var/log/pgbackrest` for the commands run (e.g. `/var/log/pgbackrest/mystanza_backup.log`)

8. Describe the issue:
Restoring a backup in one of the standby servers returns an error

```
P00  ERROR: [029]: raised from local-13 protocol: zst error: [-10] Unknown frame descriptor
                                    [FormatError] on retry after 335ms: [same message]
                                    [FormatError] on retry after 15363ms: [same message]
```

Attached is the restore log in DEBUG level and MANIFEST files for all the backups in that history

**Comment by fmbiete** [comment]
[manifests.tgz](https://github.com/pgbackrest/pgbackrest/files/12699780/manifests.tgz)
[server-restore.log](https://github.com/pgbackrest/pgbackrest/files/12699786/server-restore.log)
[stanza_info.txt](https://github.com/pgbackrest/pgbackrest/files/12699788/stanza_info.txt)
[pgbackrest_conf.txt](https://github.com/pgbackrest/pgbackrest/files/12699792/pgbackrest_conf.txt)

**Comment by fmbiete** [comment]
Hi David,
sorry, a colleague delete some of the backup files required for this restore and I cannot restore it anymore. if the information before is not enough, we cannot provide additional.

**Comment by dwsteele** [comment]
All these backups were made with 2.46 (rather than 2.47 as implied in the title), which has a known issue matching this case exactly. See https://github.com/pgbackrest/pgbackrest/pull/2123. This issue was fixed in 2.47.
